### DISCUSSION TITLE
GPU instructions on Windows 10

### DISCUSSION THREAD

**Discussion by dts350z** [JeffreyCA/spleeter-web#88] [General]
The (GPU) instructions didn't work for me on Windows 10, mainly I guess because Microsoft has changed the insiders program with the launch of windows 11 preview. My machine doesn't qualify for windows 11 (and I don't want windows 11) so it won't let me have access to the dev channel.

IMO this needs to decoupled from requiring you to be on the bleeding edge and at the mercy of whatever Microsoft throws at you.

I have docker and WSL 2.0 and Ubuntu installed, along with CUDA drivers however I get:

$ sudo docker run --rm --gpus all nvidia/cuda:11.0-base nvidia-smi

docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: Running hook #0:: error running hook: exit status 1, stdout: , stderr: nvidia-container-cli: initialization error: driver error: failed to process request: unknown.

**Comment by JeffreyCA** [comment]
Hi, I understand your point. It doesn't look like there's a way to do this using stable version of Windows at the moment. I also don't have a compatible machine to test out that scenario.

Maybe these issues could help:
https://github.com/NVIDIA/nvidia-docker/issues/1492
https://github.com/NVIDIA/nvidia-docker/issues/1409
